src/python/pants/engine/fs.py

```diff
@@ -120,7 +120,10 @@ class DigestContents(Collection[FileContent]):
     """
 
 
-class DigestEntries(Collection[Union[FileEntry, SymlinkEntry, Directory]]):
+DigestEntry = Union[FileEntry, SymlinkEntry, Directory]
```

Comment:
Why this change? Seems fine but I am trying to recall if I intentionally did not introduce such a type alias back when I wrote this rule.

Reply:
Oh, this is most likely a leftover, I originally made this change because I was handling some digest entries and wanted a handier type to deal with them. This is not needed anymore.